bundles/org.openhab.binding.withings/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="withings" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Withings Binding</name>
+	<description>The Withings binding integrates devices from Withings for example scales</description>
+	<author>Sven Strohschein</author>
+
+	<config-description/>
```

Comment:
What's the purpose of this tag?

Reply:
The OH build breaks when I remove this line. It seems to be required by the xml scheme even when it's empty... This is the log entry of the build when I remove it: "[ERROR] OH-INF.config.config.xml:[8]
The content of element config-description:config-descriptions is not complete. One of {config-description} is expected."

Reply:
solved / removed